README.md

```diff
@@ -1 +1,79 @@
-# CommunicationService
\ No newline at end of file
+# communication-service
+This is the communication of the Lisk delegate market project. It was made as a groupproject for Fontys. 
+
+It was made with .NET Core 3.1 in combination with SignalR for the Lisk Delegate Market.
+
+The other services can be found [here](https://github.com/S65-2-project). If you want to see the live product it can be found [here](https://delegate-market.nl).
+
+Via this service the user can live chat with another user to negiote deals in the application.
+
+## External sources
+To run this project you will need to run the following services:
+- [MongoDB](https://www.mongodb.com/re)
+- [RabbitMQ](https://www.rabbitmq.com/)
+- [Sentry](https://sentry.io)
+- [SignalR](https://dotnet.microsoft.com/apps/aspnet/signalr)
```

Comment:
Misschien dat we dit niet als external source moeten zien omdat het gewoon een library is

Reply:
Maar aan de andere kant snap ik je gedachte ook wel